students/15th/team1/suntae_kim/user/views.py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+import bcrypt
+import jwt
+
+from django.http import JsonResponse
+from django.views import View
+
+from user.models import User
+
+
+# Create your views here.
+
+class UserView(View):
+
+# 회원가입 로직 작성필요
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            # 기본 변수 선언
+#            mobile_email = data['mobile_email']
+#            fullname     = data['fullname']
+            username     = data['username']
+            password     = data['password']
+
+            # 정규 표현식
+            username_expression = "^[A-Za-z0-9_]*$"
+            password_expression = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!#%*?&]{8,18}"
+            email_expression    = "[^@]+@[^@]+\.[^@]+"
+            mobile_expression   = "(010)(-{1}\d{4}-{1}\d{4})"
```

Comment:
정규식은 상수라고 생각하시면 될 것 같습니다.

```suggestion
            # 정규 표현식
            REGEX_NAME = "^[A-Za-z0-9_]*$"
            REGEX_PASSWORD = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!#%*?&]{8,18}"
            REGEX_EMAIL    = "[^@]+@[^@]+\.[^@]+"
            REGEX_MOBILE   = "(010)(-{1}\d{4}-{1}\d{4})"
```